src/pages/widgets/answers.md

```diff
@@ -1,4 +1,5 @@
 ---
+layout: ../../layouts/WidgetLayout.astro
```

Comment:
This could have been a `pages/widgets/[id].astro` route that calls `Astro.glob` on another folder containing the MD files which wouldn't require us to specify layout for each file, but I prefer to play by the framework rules and write configurations over writing code.

Totally up for discussion of course !